contrib/build-push-ceph-container-imgs.sh

```diff
@@ -91,6 +84,14 @@ function install_docker {
 function install_podman {
   # https://tracker.ceph.com/issues/44242
   sudo dnf localinstall -y http://apt-mirror.front.sepia.ceph.com/lab-extras/8/fuse-overlayfs-0.7.6-2.0.dev.gitf1da779.el8.x86_64.rpm
```

Comment:
I guess we don't need to install this specific fuse-overlayfs anymore with the need repository as I can see 0.7.6-10.1 available.

Reply:
Oh I didn't notice.  I'll remove that.  Thanks!